src/components/backend-ai-resource-broker.ts

```diff
@@ -363,11 +363,11 @@ export default class BackendAiResourceBroker extends BackendAIPage {
    * Update virtual folder list. Also divide automount folders from general ones.
    *
    */
-  async updateVirtualFolderList() {
+   async updateVirtualFolderList(userEmail = false) {
```

Comment:
The type of default value of parameter `userEmail` doesn't seem to have a match with SDK function in [here](https://github.com/lablup/backend.ai-webui/pull/1237/files#diff-6ab6a3b84b5af19510d01f3a5c155018416e21b94b7090c2a4a22e0a1d68e33bR1421-R1429).
It's string, not boolean.